xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/GatewayBalancesRequestParams.java

```diff
@@ -0,0 +1,71 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.client.XrplRequestParams;
+import org.xrpl.xrpl4j.model.client.common.LedgerSpecifier;
+import org.xrpl.xrpl4j.model.transactions.Address;
+
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+/**
+ * Request parameters for the gateway_balances rippled method.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableGatewayBalancesRequestParams.class)
+@JsonDeserialize(as = ImmutableGatewayBalancesRequestParams.class)
+public interface GatewayBalancesRequestParams extends XrplRequestParams {
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableGatewayBalancesRequestParams.Builder}.
+   */
+  static ImmutableGatewayBalancesRequestParams.Builder builder() {
+    return ImmutableGatewayBalancesRequestParams.builder();
+  }
+
+  /**
+   * The unique {@link Address} for the account, which should be an issuer. The request assets and
+   * balances associated with the issuer account.
+   *
+   * @return The {@link Address} for the account.
+   */
+  Address account();
+
+  /**
+   * An optional set of addresses of operational accounts that should not be included in the
```

Comment:
Should we change this description for the opposite as we know the documentation on xrpl.org is incorrect?

Reply:
I feel comfortable saying it's incorrect based on the test actually. I created an issuer and standby and passed the standby in `hotwallet` and got it back in the results rather than having it excluded.